pkg/util/conditions/condition_types.go

```diff
@@ -38,3 +42,8 @@ const (
 	FetchingSecretRefSuccessReason ConditionReason = "FetchingSecretRefSuccess"
 	FetchingSecretRefFailedReason  ConditionReason = "FetchingSecretRefFailed"
 )
+
+const (
+	GarmServerNotReconciledYet ConditionMessage = "GARM server not reconciled yet"
+	DeletingEnterprise         ConditionMessage = "Deleting enterprise"
+)
```

Comment:
* `DeletingEnterprise` should be used in https://github.com/mercedes-benz/garm-operator/blob/ce290e7ccba2d1e457c2e1ce2e2c8fefdf2c9ebc/internal/controller/enterprise_controller.go#L220
* for `GarmServerNotReconciledYet` see my comments on organization, pool and repository controller